guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,5 @@
+We'd like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
+Adam Růžička,Alexey Masolov,Archana Kumari,Areyus,Bastian Schmidt,Dyrkon,Eric Helms,Evgeni Golov,Ewoud Kohl van Wijngaarden,Girija Soni,Griffin Sullivan,Ian Ballou,Jeremy Lenz,Joshua Hoblitt,Karolina Malyjurkova,Laura Hild,Leos Stejskal,Manisha Singhal,Marek Hulán,Maria Agaphontzev,Markus Bucher,Maximilian Kolb,Nikos Moumoulidis,Nofar Alfassi,Oleh Fedorenko,Partha Aji,Pat Riehecky,Patrick Creech,Romain Tartière,Ron Lavi,Samir Jha,Sebastian Bublitz,Shimon Shtein,Tim Meusel,Tony James,Trent Anderson,William Bradford Clark
```

Comment:
Didn't we change this to multiple lines? That makes the diff easier to read. I'd also at least expect spaces between the commas.